include/stout/flags/v1/flag.proto

```diff
@@ -8,7 +8,10 @@ message Flag {
   repeated string names = 1;
   repeated string deprecated_names = 2;
   string help = 3;
-  bool required = 4;
+  // Default value.
```

Comment:
Maybe comment on whether this default value is validated. (Can I have a flag of type "int32" with a default value of "banana"?)

Consider adding a unit test case that uses an invalid default value to show what happens.